e2e/metrics/suite_test.go

```diff
@@ -51,24 +52,27 @@ func setupLoadBalancer() {
 		metricsURL = fmt.Sprintf("http://%s:%d/metrics", ip, port)
 		return ip, err
 	}).ShouldNot(BeEmpty())
+
+	return
```

Comment:
Yes. When [named result parameters](https://go.dev/doc/effective_go#named-results) are used, it is.

> Because named results are initialized and **tied to an unadorned return**, they can simplify as well as clarify. [...]

Reply:
I see, thanks for clarifying.